test/onnx/test_pytorch_onnx_no_runtime.py

```diff
@@ -4,20 +4,22 @@
 
 import io
 import unittest
-from typing import Optional, Type
+from typing import Dict, Optional, Type
 
 import onnx
 
 import torch
 from torch import Tensor
-from torch.onnx import symbolic_helper
+from torch.onnx import OperatorExportTypes, symbolic_helper, utils
```

Comment:
use torch.onnx.OperatorExportTypes directly

Reply:
We should only import the test helper functions for now and build a good habit of keeping the namespace clean